tensorflow/tools/ci_build/linux/mkl/build-dev-container.sh

```diff
@@ -177,7 +180,12 @@ function checkout_tensorflow()
   # Let's make this simeple for now; we can be more fancy later
   git clone ${TF_REPO} ${TENSORFLOW_DIR}
   cd ${TENSORFLOW_DIR}
-  git checkout ${TF_BUILD_VERSION}
+  if [[ "${TF_BUILD_VERSION_IS_PR}" == "yes" ]]; then
+    git fetch origin pull/${TF_BUILD_VERSION}/head:pr-${TF_BUILD_VERSION}
```

Comment:
```suggestion
    git fetch origin pull/${TF_BUILD_VERSION}/head:pr-${TF_BUILD_VERSION}
```
This made me think the command is wrong for a second, because we expect the pull request ID after `git fetch origin pull/`. 

Please also add a comment here that `TF_BUILD_VERSION` is a PR ID in this case.